app/assets/stylesheets/_global.css.scss

```diff
@@ -28,10 +28,10 @@ label {
 
 // Form actions background change
 
-.form-actions {
-  background-color: darken(desaturate($bodyBackground, 10%), 6%);
-  border-top: 1px solid darken(desaturate($bodyBackground, 10%), 15%);
-}
+// .form-actions {
```

Comment:
I prefer the form buttons w/o the additional background so we can just delete these lines.